misc/scripts/install-local.sh

```diff
@@ -372,14 +365,14 @@ function hashcheck() {
 
 fancy_message info "Retrieving packages"
 mkdir -p "$SRCDIR"
-cd "$SRCDIR"
+cd "$SRCDIR" 2> /dev/null || ( fancy_message warn "Could not enter ${SRCDIR}" && exit 1)
```

Comment:
Add error logging

Reply:
Same for the others

Reply:
Can you suggest the correct snippet once? Then I can adapt the snippet for the others afterwards.

Reply:
Something like `error_log 1 "install $PACKAGE"`.

Reply:
Something like `error_log 1 "install $PACKAGE"`.

Reply:
okay, I'll add it in soon.

Reply:
Fixed in https://github.com/pacstall/pacstall/pull/276/commits/99e0e97dd340f72b3f73673958e5ead2e5370b67